components/api/core/api_impl.bal

```diff
@@ -133,32 +136,29 @@ public function createOrg(http:Request createOrgReq, gen:OrgCreateRequest create
     }
 }
 
-function addOrgUserMapping(string userId, string orgName, string role) returns http:Response{
-    var orgUserRes = db:insertOrgUserMapping(userId, orgName, role);
-    if (orgUserRes is error) {
-        log:printError(io:sprintf("Unexpected error occured while inserting org-user mapping. user : %s, Organization : %s", userId, orgName),
-                                err = orgUserRes);
-        return buildUnknownErrorResponse();
-    } else {
-        log:printDebug(io:sprintf("New organization \'%s\' added to REGISTRY_ORG_USER_MAPPING. Author : %s", orgName, userId));
-        return buildSuccessResponse();
-    }
-}
-
 public function getOrg(http:Request getOrgReq, string orgName) returns http:Response {
     json | error res = db:getOrganization(orgName);
+    string ERROR_MESSAGE = "Unable to fetch organization";
```

Comment:
Let's make this "const ERROR_MESSAGE". There must be optimizations if it is a const

Reply:
I think we can't make it const since it defines inside a function.